### PR TITLE
Update rubygem-foreman_kernel_care to 1.1.1

### PR DESCRIPTION
**Pull request by maccelf**
Update [foreman_kernel_care](https://github.com/maccelf/foreman_kernel_care) plugin

### CODE REVIEW

File: packages/katello/rubygem-foreman_kernel_care/rubygem-foreman_kernel_care.spec [hunk 1]
```diff
@@ -92,5 +94,8 @@ cp -a .%{gem_dir}/* \
 %{gem_instdir}/test
 
 %changelog
+* Fri Apr 01 2022 root 1.1.1-1
```
Comment (adamruzicka) [inline packages/katello/rubygem-foreman_kernel_care/rubygem-foreman_kernel_care.spec:97]: Could fill in your username and email as you did on L100?

**Review by adamruzicka — APPROVED** [review]
Looks good as far as I can tell

**Comment by ekohl** [comment]
ok to test

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
Right, now Katello is broken differently…

```
DEBUG util.py:598:  Error: Package: tfm-rubygem-katello-4.5.0-0.3.pre.master.20220426175037git9298009.el7.noarch (build)
DEBUG util.py:598:             Requires: tfm-rubygem(pulp_python_client) < 3.6.0
DEBUG util.py:598:             Installing: tfm-rubygem-pulp_python_client-3.6.0-1.el7.noarch (build)
DEBUG util.py:598:                 tfm-rubygem(pulp_python_client) = 3.6.0
```

Bad timing.

**Comment by evgeni** [comment]
[test rpm]

**Review by evgeni — APPROVED** [review]
repoclosure unrelated

**Pull request merged**